ray_provider/xcom/ray_backend.py

```diff
@@ -238,6 +281,56 @@ def on_success_callback(context):
 
                     RayBackend.get_hook().cleanup(handles=handles)
 
+    @staticmethod
+    def persist_kvstore(gcs_conn_id, gcs_bucket, keypath):
+        log.info("Persisting")
+        
+        def persist_gcs(gcs_conn_id, gcs_bucket, keypath, data):
+            from airflow.providers.google.cloud.hooks.gcs import GoogleCloudStorageHook
+            import json
+            import pickle 
+            try:
+                print("Serializing")
+                output = pickle.dumps(data)
```

Comment:
We will want to make sure this is pluggable in some way. Maybe let's first flag this off.